installation.rst

```diff
@@ -132,29 +132,16 @@ Install from source
 You can run Conan directly from source code. First, you need to install Python and
 pip.
 
-Clone (or download and unzip) the git repository and install its requirements:
+Clone (or download and unzip) the git repository and install it with:
 
 .. code-block:: bash
 
-    $ git clone https://github.com/conan-io/conan.git
-    $ cd conan
-    $ pip install -r conans/requirements.txt
+    # clone folder name matters, to avoid imports issues
+    $ git clone https://github.com/conan-io/conan.git conan_src
+    $ cd conan_src
+    $ pip install -e .
```

Comment:
does this also install the *requirements.txt*?

Reply:
it does
(it doesn't install `requirements_dev.txt`, but it does install `requirements.txt`)